cypress/integration/code-schools.spec.js

```diff
@@ -14,4 +14,31 @@ describe('code schools', function() {
     // 40 is arbitrary, but it proves that the API is working and leading to rendered content
     cy.get('[data-testid="SchoolCard"]').should('have.length.greaterThan', 40);
   });
+
+  it('should render Cincy and Tech Elevator after selecting Ohio', () => {
+    cy.wait('@codeSchools');
+    cy.get('input#react-select-state_select-input')
+      .type('Ohio', { force: true })
+      .type('{enter}');
+
+    cy.get('[data-testid="SchoolCard"]').should('have.length', 2);
```

Comment:
This is good. You're asserting that there only exists 2 SchoolCards, but we should also look more closely at the DOM of each of those cards to ensure they are indeed the two schools your test is named after!

Reply:
When you get a chance, can you look at this one. I replaced it with`cy.get('[data-testid="SchoolCard"]')
      .find('img[alt="Cincy Code IT Bootcamps logo"]', 'img[alt="Tech Elevator logo"]')
      .should('be.visible');`